cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/RatioTapChangerConversion.java

```diff
@@ -71,6 +74,7 @@ public boolean valid() {
             inRange("position", position, lowStep, highStep);
     }
 
+    @Deprecated
```

Comment:
Not necessary to deprecate the methods when the class is deprecated.

Reply:
Done